src/components/modals/insertImageModal/partials/UploadImageForm.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button, Center, Group, Image, Stack, Text } from "@mantine/core";
+import { Dropzone, FileWithPath } from "@mantine/dropzone";
+import { useMemo, useState } from "react";
+import { convertBase64 } from "../../../../lib/helpers/convertBase64";
+import { TInsertImageModalProps } from "../InsertImageModal";
+import Compressor from "compressorjs";
+
+type UploadImageFormProps = Pick<
+  TInsertImageModalProps,
+  "handleSubmit" | "handleCancel"
+>;
```

Comment:
```suggestion
type UploadImageFormProps = TInsertImageModalProps;
```

I think you missed a refactor here, `TInsertImageModalProps` has only `handleSubmit` and `handleCancel`, no need to Pick from them, or am I wrong?